jerry-core/api/jerry.c

```diff
@@ -3546,16 +3530,13 @@ jerry_get_backtrace (uint32_t max_depth) /**< depth limit of the backtrace */
 jerry_value_t
 jerry_get_resource_name (const jerry_value_t value) /**< jerry api value */
 {
-#if ENABLED (JERRY_RESOURCE_NAME)
```

Comment:
Are these macros removed? I see them in other files.

Reply:
I only removed them from here. Although I guess they should be kept to reduce code size.